stdlib/3/concurrent/futures/thread.pyi

```diff
@@ -13,3 +16,13 @@ class ThreadPoolExecutor(Executor):
                      thread_name_prefix: str = ...) -> None: ...
     else:
         def __init__(self, max_workers: Optional[int] = ...) -> None: ...
+
+
+class _WorkItem:
```

Comment:
```suggestion
class _WorkItem(Generic[_T]):
```

and import `Generic` above. Otherwise, there is no relation between the `fn` field and the argument of `__init__` (and `_T` would be redundant).